src/main/res/layout/main.xml

```diff
@@ -98,13 +98,12 @@
         </FrameLayout>
 
         <android.support.v7.widget.Toolbar
+            style="@style/simpleSelectionToolbar"
             android:id="@+id/toolbar"
             android:layout_width="fill_parent"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
             android:minHeight="?attr/actionBarSize"
-            android:background="@color/toolbar_background"
-            android:theme="@style/AppTheme"
```

Comment:
Doesn't appear to have changed anything, maybe all the relevant attributes have already been set? Maybe I should add it back in just in case I'm missing something, though...